lib/data/AbstractData.php

```diff
@@ -10,12 +10,14 @@
  * @version   0.22
  */
 
+namespace PrivateBin\data;
```

Comment:
I think namespace should have a captital letter. So use `PrivateBin\Data`.
Have a look at these examples: http://www.php-fig.org/psr/psr-4/#3-examples

The path can still be lowercase, but the namespace needs to start with a capital letter.